sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -221,6 +226,10 @@ public void deleteDigitalTwin(String digitalTwinId)
         return digitalTwinsAsyncClient.deleteDigitalTwinWithResponse(digitalTwinId, options, context).block();
     }
 
+    //==================================================================================================================================================
+    // Relationship APIs
```

Comment:
While you are sprucing this up, we can actually do region sections in java, too.

```java
//region Models APIs
...
//endregion Models APIs
```

Intellij will let you collapse the code just like how C# does

Reply:
much better.